src/fs-helpers.ts

```diff
@@ -39,21 +51,40 @@ export function jsonFile<T>(path: string, obj: T) {
   };
   return file;
 }
+function cloneFile<T extends File | BaseFile>(f: T): T {
```

Comment:
Given that the only place this is used the argument we pass is a `File`, do we need `BaseFile`? It actually looks to me like this could accept `File` and return `File`, no need for the generic.

Reply:
Yeah, I'd been thinking that someone might be passing in `BaseFile` instances that they built themselves, like `fixture.add({path: '/foo.txt', contents: 'foo!'});`

But probably nobody's ever gonna want to do that, or if they do they can just tag as `type: 'string'`.  I'll make the change.